2020-an-environment-for-sustainable-research-software-in-germany-and-beyond/manuscript.md

```diff
@@ -0,0 +1,2021 @@
+enable and promote synergies, it is important to allocate resources for
+research software development and to build **communities**, as described
+in (Katz et al. 2019).
+
+Challenges
+----------
+
+We are currently facing a **lack of awareness** for the importance of
+research software as discussed in the section on p. . Moreover, there is
+**little recognition** for the efforts put into software development and
+maintenance. In consequence, software development in academic settings
+is mostly considered as a means to an end and sustainability is often
+not considered in project planning and grant proposals and contributes
+little to progressing research careers (Science Guide 2019; “San
+Francisco Declaration on Research Assessment” 2012). The main challenge
+here is the continued use of metrics that primarily leverage
+traditionally published articles and article citation numbers.
+
+In academia, developers of research software are typically domain
+researchers, and in particular if new areas are explored, the software
+development process itself has research character. Obviously, developing
+research software requires not only domain knowledge but also software
+development skills, and the researchers leading the software development
+process are often domain experts with substantial software development
+experience. However, the current academic system in Germany does not
+provide a defined **RSE role**. Limited-term positions are, at least
+currently within the main German academic system, often effectively the
+end of their career path, sometimes even a dead end. The challenge here
+is the lack of available permanent positions within the non-professorial
+academic faculty (“Mittelbau”) in Germany, as well as the already
+mentioned lack of recognition for efforts concerning research software
+for faculty appointments within domain sciences.
+
+In order to develop sustainable software, researchers need to have the
+**skills and expertise** to build software that is easy to maintain and
+extend (Carver, Hong, and Thiruvathukal 2016). However, most of the
+researchers are self-taught developers (Wilson et al. 2014; Philippe et
+al. 2019). Ideally, these skills have to be built into the domain
+science curricula, which could generally be done in two different ways
+(or a combination of them). One obvious solution attempt are additional
+courses that focus on these topics. The main challenge here is to decide
+which other topic(s) to possibly drop due to the limited volume of any
+given curriculum. A different approach is to incorporate
+software-related topics into existing domain science courses. While this
+would provide the benefit of show-casing the usage of specific software
+skills directly within the domain science, the challenge here is the
+amount of work necessary to change existing lecture material, let alone
+the need of the lecturers to acquire those skills themselves in the
+first place.
```

Comment:
Doing this introduces the challenge of educating people twice. First in the domain and secondly for software engineering. SE itself is a craftsmanship. This way you create double headed personal. Also with personal rotation, you loose all the SE education efforts regularly.

Reply:
We do mention this as alternative approaches, to avoid educating people twice. Concerning the rotation: you do loose them anyway, that problem is independent of the issue here - apart from the expectation that the better they are educated concerning SE, the easier you might loose them because their value for industry will be a lot higher too, not just for science. But that's a price we cannot avoid.